scripts/testing/run_e2e_module_tests.sh

```diff
@@ -109,6 +110,27 @@ fi
 
 echo -e "\n--- Deprovisioning safety measures work"
 
+echo -e "\n--- Checking module resources reconciliation when BtpOperator CR is in Deleting state"
+
+echo "Deleting ${SAP_BTP_OPERATOR_DEPLOYMENT_NAME} deployment"
+kubectl delete -n kyma-system deployment/${SAP_BTP_OPERATOR_DEPLOYMENT_NAME}
+
+echo -e "\n--- Triggering reconciliation by annotating BtpOperator CR"
+kubectl annotate --overwrite -f ${YAML_DIR}/e2e-test-btpoperator.yaml last-manual-reconciliation-timestamp="$(date -u -Iseconds)"
+
+echo -e "\n--- Waiting for reconciliation (${SAP_BTP_OPERATOR_DEPLOYMENT_NAME} deployment existence)"
```

Comment:
Can we assert that deployment is removed at this point?

Reply:
Yes, we can! I will add the assertion.

Reply:
Added.